PublicMapping/districtbuilder
# Display FIPS code for blocks and blockgroups

## Overview

Does two things:
- Updates the `process-geojson` command to insert FIPS code information into the `name` field for Blocks and Blockgroups so that they display the appropriate code when the user hovers over them.
- Adds a `bulk-reprocess-regions` command that allows the user to reprocess a set of regions as a batch.

### Checklist


### Demo

<img width="407" alt="Screen Shot 2021-09-28 at 1 22 21 PM" src="https://user-images.githubusercontent.com/447977/135135805-3ade9660-5db0-43e7-bcf3-004ce2006c9b.png">


### Notes

- I haven't actually run the reprocessing script yet; I can do that after review or we can make a new card since we're butting up against the sprint boundary.
- There are plenty of helpful features that could be added to the `bulk-reprocess-regions` command that I didn't add in the interest of time. But I expect that it may be helpful to improve handling of errors so that the rest of a batch can continue even if one region fails, and potentially a default set of processing parameters so that the arguments to `process-geojson` don't have to be explicitly spelled out in every case.

## Testing Instructions

- I've uploaded the most recent input GeoJSON files for every state into the data bucket, under `/geojsons`. Pick one of those for a region that you have active in your development environment and download it to somewhere in your `src/manage/data` tree so that the `manage` container can access it.
- Configure a bulk reprocessing config file for the region you downloaded by following the directions in the manage README. Confirm that instructions are clear enough to allow you to get everything working relatively easily.
- Run the `bulk-reprocess-regions` command with the config file you created. Confirm that it works successfully and that files are updated on S3.
- Start or restart your application server and load a project that uses the region that you just updated. Confirm that blocks and blockgroups are labeled with the appropriate FIPS code when you mouse over them. Confirm that the FIPS codes nest appropriately.

Closes #568
